services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None, actors_ids: list[int] = None
+) -> QuerySet:
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            actors__in=actors_ids, genres__in=genres_ids
+        )
+    elif not (genres_ids) and actors_ids:
+        return Movie.objects.filter(actors__in=actors_ids)
+    elif genres_ids and not (actors_ids):
+        return Movie.objects.filter(genres__in=genres_ids)
+    return Movie.objects.all()
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
Using `get()` without handling exceptions can lead to `DoesNotExist` errors if the movie ID is not found. Consider wrapping this in a try-except block to handle such cases gracefully.